index.html

```diff
@@ -5,15 +5,36 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">

```

Comment:
you are not supposed to edit this file in this issue

Reply:
i apologize about that , i have returned it to the initial state..